packages/oas-to-har/src/index.js

```diff
@@ -14,11 +15,17 @@ function formatter(values, param, type, onlyIfExists) {
     return undefined;
   }
 
-  if (param.required && param.example) {
-    return param.example;
+  if (param.required && param.schema && param.schema.default) {
```

Comment:
Making this change because we shouldn't be using examples in compiled HAR files we make requests with. If there's a value set in the `default` we should use that instead. Examples should only be for reference, not actual use.